Cirumflex_Combined.ino

```diff
@@ -1,15 +1,25 @@
+// Uncomment to turn on, Comment to turn off debug() output:
+//#define DEBUG
+#include "debug.h"
+
 //Fixed colors set as globals. 03-Pattern fades between them.
 const int RANDOMnoisePIN = A5; // for random seed
-const int BRIGHTNESS = 1;      // Value pot pin number
-const int SLOWNESS = 3;        // Pot pin number; 0 is fast, 1023 is slow.
+const int BRIGHTNESS = -1;      // e/g. A0. Value pot pin number; use -1 for no pot
+const float BRIGHTNESS_OVERRIDE = 0.2; // e.g. A0. if pin == -1, use this as brightness
+const int SLOWNESS = -1;        // e.g. A1. Pot pin number; 0 is fast, 1023 is slow.
+const float SLOWNESS_OVERRIDE = 1; // if pin == -1, use this as slowness
```

Comment:
All that allows you to have/not-have the pots. Use -1 for the pins to "not have". Then you get the override as the value of the pot. Note, the slowness is the actuall POT value, not a delay value..